Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ApplicationManagment/ApplicationNode.tsx

```diff
@@ -87,7 +87,7 @@ const ByApplicationNode: Application.Types.iByComponent = (props) => {
         <>
             <LoadingScreen Show={status === 'loading'} />
             <div style={{ width: '100%', height: '100%' }}>
-                <SearchBar<SystemCenter.Types.Setting> CollumnList={searchFields} SetFilter={(flds) => dispatch(ApplicationNodeSlice.DBSearch({ filter: flds, sortField, ascending }))}
+                <SearchBar<SystemCenter.Types.Setting> StorageID={"SystemCenter.ApplicationNode.Search"} CollumnList={searchFields} SetFilter={(flds) => dispatch(ApplicationNodeSlice.DBSearch({ filter: flds, sortField, ascending }))}
```

Comment:
Typoeing for `SearchBar<SystemCenter.Types.Setting>` is not right. This should be `SearchBar<SApplicationNode>` or similar